detection_rules/rule_loader.py

```diff
@@ -234,9 +234,14 @@ def filter_rules(rules, metadata_field, value):
     return [rule for rule in rules if rule.metadata.get(metadata_field, '') == value]
 
 
-def get_production_rules(verbose=False):
+def get_production_rules(verbose: bool = False, include_deprecated: bool = False) -> List[Rule]:
```

Comment:
i think it's okay to let the types be inferred. your choice
```suggestion
def get_production_rules(verbose=False, include_deprecated=False) -> List[Rule]:
```